nautobot/extras/filters/mixins.py

```diff
@@ -92,7 +97,8 @@ def _generate_custom_field_lookup_expression_filters(cls, filter_name, custom_fi
         custom_field_type_to_filter_map = {
             CustomFieldTypeChoices.TYPE_DATE: CustomFieldMultiValueDateFilter,
             CustomFieldTypeChoices.TYPE_INTEGER: CustomFieldMultiValueNumberFilter,
-            CustomFieldTypeChoices.TYPE_SELECT: CustomFieldMultiValueCharFilter,
+            CustomFieldTypeChoices.TYPE_SELECT: CustomFieldMultiSelectFilter,
```

Comment:
Should this be `CustomFieldSelectFilter`? Or was the existing use of `CustomFieldMultiValueCharFilter` incorrect for some reason?